config/samples/operator_v1alpha1_manifest.yaml

```diff
@@ -18,12 +18,12 @@ spec:
     spec:
       randomkey: samplevalue
   installs:
-#    - source:
-#        name: kyma.project.io/module/kyma-redis
-#        repo: europe-west3-docker.pkg.dev/sap-kyma-jellyfish-dev/operator-demo
-#        ref: sha256:cd98ce9b440fef2e1d11fe382ccf6fab971f516efa9ee174d73209777ed318b5
-#        type: oci-ref
-#      name: kyma-redis-operator
+    - source:
```

Comment:
this manifest is used in samples, are we sure this is okay to remove comments here?

Reply:
committed unintentionally, it's used for me locally to test if manifest convert works or not. It indeed rejects this manifest cr due to `ErrSingleInstallOnly`